steve/util.py

```diff
@@ -216,18 +391,21 @@ def get_project_config_file_name():
     return SteveConfig.file_name
 
 
-def create_project_config_file(dir_path):
+def create_project_config_file(dir_path, cfg=None):
```

Comment:
That seems weird. Amongst other things, it breaks Python's self-documenting abilities.